.circleci/config.yml

```diff
@@ -135,6 +136,27 @@ jobs:
           name: All nightly CPU tests
           command: python setup.py test -s tests.suites.nightly_cpu -q
 
+  deploy_website:
+    executor: standard_cpu
+    working_directory: ~/ParlAI
+    steps:
+      - checkout
+      - setup
+      - installdeps
+      - installtorchcpu
```

Comment:
Curious - is the `installtorchcpu` job really a dependency for this?

Reply:
Yeah I thought that was funny too, but the docs need it because they torch_agent imports it.